solr/src/test/java/akka/stream/alpakka/solr/SolrTest.java

```diff
@@ -322,13 +351,136 @@ public void testKafkaExample() throws Exception {
         result.stream().sorted().collect(Collectors.toList()));
   }
 
+  @Test
+  public void deleteDocuments() throws Exception {
+    // Copy collection1 to collection2 through document stream
+    createCollection("collection7"); // create a new collection
+    TupleStream stream = getTupleStream("collection1");
+
+    // #run-document
+    SolrUpdateSettings settings = SolrUpdateSettings.create().withCommitWithin(5);
+    CompletionStage<Done> f1 =
+        SolrSource.fromTupleStream(stream)
+            .map(
+                tuple -> {
+                  Book book = tupleToBook.apply(tuple);
+                  SolrInputDocument doc = bookToDoc.apply(book);
+                  List<IncomingMessage<SolrInputDocument, NotUsed>> list = new ArrayList<>();
+                  list.add(IncomingUpdateMessage.create(doc));
+                  return list;
+                })
+            .runWith(
+                SolrSink.documents("collection7", settings, cluster.getSolrClient()), materializer);
+    // #run-document
+
+    f1.toCompletableFuture().get();
+
+    TupleStream stream2 = getTupleStream("collection7");
+
+    CompletionStage<Done> res2 =
+        SolrSource.fromTupleStream(stream2)
+            .map(
+                t -> {
+                  List<IncomingMessage<SolrInputDocument, NotUsed>> list = new ArrayList<>();
+                  list.add(IncomingDeleteMessage.create(tupleToBook.apply(t).title));
+                  return list;
+                })
+            .runWith(
+                SolrSink.documents("collection7", settings, cluster.getSolrClient()), materializer);
+
+    res2.toCompletableFuture().get();
+
+    TupleStream stream3 = getTupleStream("collection7");
+
+    CompletionStage<List<String>> res3 =
+        SolrSource.fromTupleStream(stream3)
+            .map(t -> tupleToBook.apply(t).title)
+            .runWith(Sink.seq(), materializer);
+
+    List<String> result = new ArrayList<>(res3.toCompletableFuture().get());
+
+    List<String> expect = Arrays.asList();
+
+    assertEquals(expect, result);
+  }
+
+  @Test
+  public void atomicUpdateDocuments() throws Exception {
+    // Copy collection1 to collection2 through document stream
+    createCollection("collection8"); // create a new collection
+    TupleStream stream = getTupleStream("collection1");
+
+    // #run-document
```

Comment:
Did you check the generated documentation? It looks like the `#run-document` tag is used for different things.

Reply:
No